remote-api/src/test/java/org/alfresco/rest/api/tests/ProbeApiTest.java

```diff
@@ -128,32 +136,28 @@ private void assertResponse(String probeName, Boolean ready, String expected, in
     public void testProbes() throws Exception
     {
         // Live first
-        assertResponse(LIVE, ERROR, "liveProbe: Failure - Tested", 503);
-        assertResponse(LIVE, null, "liveProbe: Failure - No test", 503); // Need to wait 10 seconds.
-        assertResponse(LIVE, null, "liveProbe: Failure - No test", 503);
-        assertResponse(READY, null, "readyProbe: Failure - No test", 503);
+        assertResponse(LIVE, OK, "liveProbe: Success - Tested", 200);
```

Comment:
I don't understand how this test works. Please can we split it up into lot of simpler unit tests?